kafka_consumer/tests/docker/noauth/docker-compose.yaml

```diff
@@ -1,15 +1,25 @@
-version: '3.5'
+version: '3.9'
 
 services:
   zookeeper:
```

Comment:
Do we need this anymore?

Reply:
Yes, we still need to have Zookeeper since Kafka still uses Zookeeper to manage the clusters. It's just that the check no longer needs to query Zookeeper itself to get the Kafka metrics (starting in Kafka 0.10).